packages/inquirer-common/jest.config.js

```diff
@@ -0,0 +1,6 @@
+const config = require('../../jest.base');
+config.snapshotFormat = {
+    escapeString: false,
+    printBasicPrototype: false
+};
```

Comment:
Do we need these? These were just set in tools-suite for backward compatibility from what I recall.

Reply:
OK Ill check, we dont want escaped snapshots right?